README.md

```diff
@@ -2319,6 +2319,41 @@ ANSWERS.addComponent('MyCustomComponent', {
 });
 ```
 
+# Custom Partials
```

Comment:
Can we add this to the README table of contents? It's located at the top of the README.

Since this section is similar to the "Template Helpers" section, what do you think of making an H1 section called "Custom Rendering", and inside that we have H2 sections for "Custom Partials" and "Template Helpers"?

Reply:
Yeah, I was also thinking it might be nice to group them. Would the H1 title be confusing since there is also a "Using a Custom Renderer" section that might look similar?

Reply:
Maybe we have the H1 be "Extending the built-in renderer"? That might be distinct enough from "Using a custom renderer"